foxy/distribution.yaml

```diff
@@ -1899,6 +1902,7 @@ repositories:
       url: https://gitlab.com/micro-ROS/ros_tracing/ros2_tracing-release.git
       version: 1.0.1-2
     source:
+      test_abi: true
```

Comment:
Does it matter that this repository is hosted on GitLab instead of GitHub?

Reply:
The abi checking uses `auto-abi` tool and in the [ROS buildfarm it does work with](https://github.com/ros-infrastructure/ros_buildfarm/blob/master/scripts/devel/build_and_install.py#L49-L50):
 * released packages
 * install directory in the workspace

gitlab/github should not impact in how it works, as far as I can say.